kotlinx-coroutines-core/common/src/CoroutineDispatcher.kt

```diff
@@ -27,46 +27,30 @@ import kotlin.coroutines.*
  *
  * This class ensures that debugging facilities in [newCoroutineContext] function work properly.
  */
-public abstract class CoroutineDispatcher :
-    AbstractCoroutineContextElement(ContinuationInterceptor), ContinuationInterceptor {
+public abstract class CoroutineDispatcher
+    : AbstractCoroutineContextElement(ContinuationInterceptor), ContinuationInterceptor {
```

Comment:
Style (quote): "For classes with a long supertype list, _put a line break after the colon_ "